conda/recipes/ucx-py/meta.yaml

```diff
@@ -21,18 +25,17 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - cython
-    - pip
     - python
-    - tomli  # [py<311]
+    - pip
     - ucx
-    - versioneer >=0.24
+    {% for r in data.get("build-system", {}).get("requires", []) %}
+    - {{ r }}
+    {% endfor %}
   run:
-    - numpy
     - python
-    - ucx
```

Comment:
Would also note that [`ucx` has `run_exports`]( https://github.com/conda-forge/ucx-split-feedstock/blob/dd05d5af3a6c4902093ecff980564b6605633124/recipe/meta.yaml#L38-L39 ) as of `1.12.1` ( https://github.com/conda-forge/ucx-split-feedstock/pull/92 ). As we are using `ucx` version `1.13.0`, we can rely on `run_exports` to add `ucx` as a dependency in `run` (with proper version constraints) based on what was used in `host`